1.10/tutorials/dcos-101/app1.md

```diff
@@ -10,30 +10,30 @@ menu_order: 3
+  * Have a look at the [app definition](https://raw.githubusercontent.com/joerg84/dcos-101/master/app1/app1.json). The app definition is the configuration which Marathon will use to deploy and manage the application. This app definition will download the python script and then run it inside the `mesosphere/dcos-101` Docker container.
+  * Add the app to Marathon using the app definition: `dcos marathon app add https://raw.githubusercontent.com/joerg84/dcos-101/master/app1/app1.json`
+* Check that app1 is running:
     * By looking at all DC/OS tasks: `dcos task`. Here you should look at the state this task is currently in, which probably is either *S*taging or *R*unning.
     * By looking at all Marathon apps: `dcos marathon app list`.
-    * By checking the logs: `dcos task log app1`. Here you should see on which node and port app1 is running. This might vary between different runs and even during the lifetime of the app.
+    * By checking the logs: `dcos task log app1`. Here you should see on which node and port app1 is running, and the output from the app showing the number of keys in Redis. The node and ports  might vary between different runs and even during the lifetime of the app, depending on events in the cluster.
 
 # Outcome
 You have deployed your first app inside a Docker container using Marathon.
-You verified the app is running and successfully connected to the previously deployed redis service.
+You verified the app is running and successfully connected to the previously deployed Redis service.
 
 # Deep Dive
-You have just deployed your first app using [Marathon](https://mesosphere.github.io/marathon/) directly. Also note that the redis service is running internally via Marathon.
-Marathon is also referred to as the init system of DC/OS, as its main job to support long running services.
+You have just deployed your first app using [Marathon](https://mesosphere.github.io/marathon/) directly. Also note that the Redis service itself is running via Marathon.
+Marathon is referred to as the init system of DC/OS, as its main job is to support long running services.
 Marathon also allows for scaling or uninstalling of applications.
-As such you have multiple options to deploy and maintain apps on Marathon besides the DC/OS GUI:
+There are multiple options to deploy and maintain apps on Marathon besides the DC/OS GUI:
 
-* DC/OS CLI: You have just used that option to deploy your app1. To get more information check `dcos marathon app --help`.
-* HTTP endpoints: Marathon also comes with an extensive [REST API](https://mesosphere.github.io/marathon/docs/generated/api.html) which can also be used to deploy apps
+* DC/OS CLI: You have just used this option to deploy your app1. To get more information on the marathon CLI use `dcos marathon app --help`.
```

Comment:
With changes above, there's now some inconsistency with how we refer to the app ("app" vs. "app1"). Here I'd say either "your app" or simply "app1".